code/game/objects/items/cigarettes.dm

```diff
@@ -142,7 +142,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	w_class = WEIGHT_CLASS_TINY
 	slot_flags = ITEM_SLOT_MASK
 	grind_results = list()
-	heat = 1000
+	heat = 415 //BUBBER EDIT CHANGE - Limits heat exposure to prevent condensing THC or turning space drugs to fentanyl. Original: 1000.
```

Comment:
Since you modularized this you can remove this change
```suggestion
	heat = 1000
```

Reply:
Ack I swear to god I removed this, thanks a ton.